EffectPackages.ini

```diff
@@ -340,3 +340,12 @@ TextureInstallPath=.\reshade-shaders\Textures
 DownloadUrl=https://github.com/nullfrctl/reshade-shaders/archive/refs/heads/main.zip
 RepositoryUrl=https://github.com/nullfrctl/reshade-shaders
 EffectFiles=blur.fx,hyperblur.fx,anamorpho.fx
+
+[37]
+PackageName=Reshade Simple HDR Shaders by MaxG3D
+PackageDescription=Reshade HDR-Compatible shaders that focus on eye-candy effects and basic adjustments
+InstallPath=.\reshade-shaders\Shaders\MaxG3D
+TextureInstallPath=.\reshade-shaders\Textures
+DownloadUrl=https://github.com/MaxG2D/ReshadeSimpleHDRShaders/archive/refs/heads/main.zip
+RepositoryUrl=https://github.com/MaxG2D/ReshadeSimpleHDRShaders
+EffectFiles=HDRBloom.fx, HDRSaturation.fx, HDRMotionBlur.fx
```

Comment:
Just need to get rid of those spaces after the commas, since the setup tool can't parse those, and we are good to go

Reply:
Done